src/aqm_cap.F90

```diff
@@ -60,6 +60,9 @@ module AQM
     /)
 
   private
+
+  real(8), parameter :: pi = 3.1415926535897931
+  real(8), parameter :: rad2deg = 180./pi
```

Comment:
These parameters are used to modify `coord` array which has `ESMF_KIND_R8` kind, so I would declare these two parameters with the same kind value. Besides literal `8` in `real(8)` is not portable.